independent-projects/resteasy-reactive/pom.xml

```diff
@@ -49,7 +49,7 @@
         <!-- Versions -->
         <jakarta.enterprise.cdi-api.version>4.0.1</jakarta.enterprise.cdi-api.version>
         <jandex.version>3.1.5</jandex.version>
-        <bytebuddy.version>1.12.12</bytebuddy.version>
+        <bytebuddy.version>1.14.6</bytebuddy.version>
```

Comment:
This should be aligned with the one used by the Quarkus BOM, required by Hibernate ORM.

Reply:
updated right now to 1.14.7, that is defined in quarkus BOM